packages/wallet-ui/src/utils/nodeTokenClient.ts

```diff
@@ -8,26 +12,69 @@ export enum ShortTokenNetworksName {
   homestead = "eth"
 }
 
+export const getFormattedBalanceFrom = (
+  tokenAddresses: AssetType[],
+  index: number = 0,
+  from: "walletBalance" | "counterfactualBalance" = "counterfactualBalance"
+): string => {
+  return formatEther(
+    (tokenAddresses[index] && tokenAddresses[index][from]) || Zero
+  );
+};
+
+export const getUserWalletBalances = async (
+  provider: Web3Provider,
+  walletAddress: string,
+  tokens: AssetType[]
+): Promise<any> => {
+  tokens[0].walletBalance = await provider.getBalance(walletAddress);
+  for (let index = 1; index < tokens.length; index = index + 1) {
+    const token = tokens[index];
+    try {
+      const response = await fetch(
+        `https://kovan.etherscan.io/api?module=account&action=tokenbalance&contractaddress=${token.tokenAddress}&address=${walletAddress}&tag=latest&apikey=${process.env.ETHERSCAN_API_KEY}`
```

Comment:
For React to import env vars, you must prefix them with `REACT_APP_`: https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables

Reply:
fixed!